<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserChargeWindow</class>
 <widget class="QWidget" name="UserChargeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>甜鱼: 用户充值界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    border: none;
    border-radius: 4px;
	padding: 4px;
	margin: 4px; 
    background-color: rgb(54, 125, 221);
    color: white;
}
QLineEdit{
	border: 1px solid black;
	border-radius: 4px;
	padding: 4px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>甜鱼二手交易平台</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>用户充值界面</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>充值码:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="charge_code_text">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>充值金额:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="money_text"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="postBtn">
       <property name="text">
        <string>提交信息</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
